markdown/3.x/en/documentation/routing.md

```diff
@@ -35,6 +35,14 @@ function isSearchPage() {
 }
 ```
 
+#### NOTE
+
+Please do not use `search-immediate` on non-search pages or you will be immediately redirected to the search page.
+
+#### `before-search` event
+
+`before-search` event that is emitted by `ff-searchbox` can be also used to detect certain search events in order to perform redirects. More specifically, this event will occur for all searches that involve the searchbox, but will not occur for ASN, paging, sorting etc. requests. This might come in handy on category pages where entering a new query should trigger a new search that is not be restricted by the current category filter.
```

Comment:
`is not be` sounds not quite right. Is there a word missing or too much?

Reply:
What's wrong with it?

Reply:
Would have expected something like.
`that is not restricted`
`that shall not be restricted`
`that is not to be restricted`

Reply:
Ah, right, I have no clue how `be` got there :D